### ISSUE TITLE
Chart starts rendering from the right side of the screen

### ISSUE THREAD

**Issue by markzolotoy** [plotly/plotly.js#3198]
I am noticing that after updating to the latest version a chart noticeably starts appearing on the screen from the right side of the provided container. I am exclusively working with line charts and dont know if it's happening with other types. I am charting about 800 points at a time.

**Comment by antoinerg** [comment]
Hello @markzolotoy and thank you for reporting your issue! Could you provide us with a Codepen so we can better identify the problem you're having?

Thank you again :jack_o_lantern:

**Comment by markzolotoy** [comment]
Not sure I can provide some code. But here is a picture:
![image](https://user-images.githubusercontent.com/7363944/47856799-0e63ad80-ddb6-11e8-978b-32d650c37444.png)
It then moves very quickly to the left. Much more noticeable in IE11.

**Comment by antoinerg** [comment]
Hello @markzolotoy, I'm not sure I understand the issue you're having. Is the problem that after a call to `Plotly.react`, the figure moves to the left of the page? If it is, I have never seen anything like it. Also, does it change?

Anyway, in order for me to debug, I first need to reproduce the problem over here on my machine. Can you send me the javascript snippet you use to do the plotting as well as the context of the figure in the page (the HTML/CSS describing the page). Providing a Codepen is really the best way to help me help you :)

**Comment by markzolotoy** [comment]
That started with the latest release. I am not sure I can provide an example that would illustrate the problem. I am not doing anything special when it comes to creating a chart. It might be my environment that does that. But again, it didn't do it with the previous version. But even it's on my end creating an example that would mimic my environment is very challenging. I still might trying doing it but it's not simple.

**Comment by markzolotoy** [comment]
Did not notice some technical questions. Yes, the problem is that after a call to Plotly.react. Not sure if it changes the size. In IE it is more obvious. It stays it the position shown on the screenshot for a moment and quickly allocates the rest of the space provided.

**Comment by antoinerg** [comment]
That definitely sounds like a bug although it's hard for me to find the culprit :thinking: Does it happen with all trace types?

**Comment by markzolotoy** [comment]
I only do line charts. Seems to me that it is some kind of heavy processing problem. It is noticeable even in Chrome which is a very well performing Browser. But in IE which is just the opposite it's seen much better. My QA reported about it right away and they are running Chrome, although their boxes are of a low end computers. That's why I am saying it might be a long processing issue.

**Comment by markzolotoy** [comment]
I have switched back to the previous version and it is definitely not doing this anymore. After doing this I looked closer to how it's working in IE. I have a progress bar that spins till data is arriving from the backend. In Chrome it takes no time between when it stops spinning and a chart goes on screen. But in IE it is a clear pause. All that means it is a rendering process that causes charting rendering in two noticeable steps. again, prev. version doesn't have this problem.
